eng/ILLink.Substitutions.Resources.template

```diff
@@ -0,0 +1,9 @@
+<linker>
+  <assembly fullname="{AssemblyName}">
+    <!-- System.Resources.UseResourceKeys removes resource strings and instead uses the resource key as the exception message -->
+    <resource name="{StringResourcesName}.resources" action="remove" feature="System.Resources.UseResourceKeys" featurevalue="true" />
```

Comment:
To simplify you can put the `feature` and `featurevalue` attributes on the `assembly` element - anything "under" it will be conditional (no need to repeat the feature switch then).